### ISSUE TITLE
Pressure not pulling from Bloomsky API

### ISSUE THREAD

**Issue by cmplant3210** [gjr80/weewx-bloomsky#4]
In the default sensor map in the driver file, i noticed that i had to change the *.Data.pressure to a capital P for it to work, i also had to change the barometer to pressure, so that weewx would then calculate the barometer correctly. 

DEFAULT_SENSOR_MAP = 'barometer':       '*.Data.pressure'

Changed it to = 'pressure':             '*.Data.Pressure'

Should this be correct?

**Comment by gjr80** [comment]
You are correct, case matters and it should be `Data.Pressure`. Not sure what happened or why, the previous release was correct.

Fixed at commit 3302a0ba2befe03ba5f79f870041a4af6fb4a31a and in [release v2.0.1](https://github.com/gjr80/weewx-bloomsky/releases/tag/v2.0.1).
